### PR TITLE
Add ECS Experimentation Configuration Service url

### PR DESCRIPTION
**Pull request by juan-arias**
If customers don't have access to this service's url, it will affect Microsoft's ability to respond to a severe bug in a timely manner.

### CODE REVIEW

**Comment by Court72** [comment]
@henrymbuguakiarie

Can you review the proposed changes? 

When the changes are ready for publication, add a `#sign-off` comment to signal that the PR is ready for the review team to merge. 

#label:"aq-pr-triaged"
@MicrosoftDocs/public-repo-pr-review-team

File: docs/identity-platform/apple-sso-plugin.md [hunk 1]
```diff
@@ -54,6 +54,7 @@ To use the Microsoft Enterprise SSO plug-in for Apple devices:
 
   - `app-site-association.cdn-apple.com`
   - `app-site-association.networking.apple`
+  - `config.edge.skype.com`
```
Comment (oldalton) [inline docs/identity-platform/apple-sso-plugin.md:57]: Can you add a note below to clarify why it is needed?

Comment (juan-arias) [reply docs/identity-platform/apple-sso-plugin.md]: Good idea, updated.

File: docs/identity-platform/apple-sso-plugin.md [hunk 2]
```diff
@@ -61,10 +61,13 @@ To use the Microsoft Enterprise SSO plug-in for Apple devices:
...
   - `login.microsoftonline.us`(*)(**)
   - `login-us.microsoftonline.com`(*)(**)
+  - `config.edge.skype.com`(***)
 
   (*) Allowing Microsoft domains is only required on operating system versions released before 2022. On the latest operating system versions, Apple relies fully on its CDN. 
 
-  (**) You only need to allow sovereign cloud domains if you rely on those in your environment. 
+  (**) You only need to allow sovereign cloud domains if you rely on those in your environment.
+
+  (***) Maintaining communications with the ECS Experimentation Configuration Service ensures that Microsoft can respond to a severe bug in a timely manner.
```
Comment (henrymbuguakiarie) [inline docs/identity-platform/apple-sso-plugin.md:70]:
```suggestion
  (***) Maintaining communications with the Experimentation Configuration Service (ECS) ensures that Microsoft can respond to a severe bug in a timely manner.
```

**Review by henrymbuguakiarie — CHANGES_REQUESTED** [review]
Hello @juan-arias,

I have a small request. Is ECS an acronym for Experimentation Configuration Service?

File: docs/identity-platform/apple-sso-plugin.md (hunk 2 above)
Comment (juan-arias) [reply docs/identity-platform/apple-sso-plugin.md]: yes, thank you for the suggestion Henry

**Review by henrymbuguakiarie — APPROVED** [review]
Hello @juan-arias,

LGTM

**Comment by henrymbuguakiarie** [comment]
#sign-off

**Pull request merged**